tests/Unit/CurrencyHelperTest.php

```diff
@@ -120,4 +121,30 @@ public function testToFormatCanDisplayUpToSixDecimals()
 
         $this->assertEquals($expected, $actual);
     }
+
+    /**
+     * Tests that it returns formatted value with minor unit symbol from fractional monetary values.
+     */
+    public function testToFormatCanDisplayMinorUnitSymbol()
+    {
+        $this->loadConfiguration()->loadServiceProvider();
+
+        $expected = '20p';
+        $actual = (new CurrencyHelper($this->app->config))->formatToMinorUnitWhenApplicable(20);
+
+        $this->assertEquals($expected, $actual);
+    }
+
+    /**
+     * Tests that it returns formatted value without minor unit symbol from fractional monetary values.
+     */
+    public function testToFormatCanDisplayMinorUnitSymbolWithoutPattern()
+    {
+        $this->loadConfiguration()->loadServiceProvider();
+
+        $expected = '€0.20';
+        $actual = (new CurrencyHelper($this->app->config))->formatToMinorUnitWhenApplicable(20, CurrencyCode::EURO, 'ie');
+
+        $this->assertEquals($expected, $actual);
+    }
```

Comment:
I think these two tests cover the new feature to the package but it would be good to test it with more cases.

Ideally we'd want to generate them randomly but I'm not sure how you'd generate the expected case without using the you added itself so probably we're best off going with a dataprovider here and hardcoding a few different cases to make the tests stronger

Reply:
What other cases did you have in mind? Bearing in mind that we also test cases on the admin-tool side. I'm not sure how much redundancy would there be + there are existing code and existing function that already has tests.

Reply:
We could spot check some more values instead of just 20 euro cents and 20 pence. Using a dataprovider that checks 5 different values for example